### ISSUE TITLE
missing packages in latest release

### ISSUE THREAD

**Issue by delfick** [zcutlip/pyonepassword#64]
looks like you need to change the setup.py to say something like

```
packages = find_packages(include="pyonepassword.*", exclude=["tests*"])
```

it seems you don't have a github action for pypi release, they make things much nicer, [for example](https://github.com/delfick/strcs/blob/main/.github/workflows/release.yml). Otherwise if you make the package locally, it seems having an egg-info folder present can confuse it slightly, so make sure one of those don't already exist if you're running `setup.py sdist`

**Comment by zcutlip** [comment]
hmm I see what happened
at [setup.py:24](https://github.com/zcutlip/pyonepassword/blob/c88268dd859a7cce6f728424b233c8746b69b239/setup.py#L24), I need to add a wildcard (as in your example): `pyonepassword.*`

What I'm not clear about is why `tox` isn't catching this. 

Int the tox env, the `pyonepassword` is definitely incomplete as you observe.

Pytest is picking up packages & modules from the current working directory rather than from tox's virtualenv

**Comment by zcutlip** [comment]
hmm it's a known problem
https://stackoverflow.com/a/55762019/17391340
https://docs.pytest.org/en/latest/explanation/goodpractices.html#tests-outside-application-code

**Comment by zcutlip** [comment]
ah...okay
we can set `pytest`'s `import-mode` to "append", which will cause directories to be added to the *end* of `sys.path` path rather than the beginning. This will cause `tox`'s installed packages to take precedent

```
[pytest]
addopts = "--import-mode=append"
```

**Comment by zcutlip** [comment]
should be fixed in #66 
Need to address GitHub action not running in #65 before cutting a release
